src/languages/stan.js

```diff
@@ -504,16 +345,38 @@ export default function(hljs) {
       $pattern: hljs.IDENT_RE,
       title: BLOCKS,
       keyword: STATEMENTS.concat(VAR_TYPES).concat(SPECIAL_FUNCTIONS),
-      built_in: FUNCTIONS
+      built_in: FUNCTIONS.concat(DISTRIBUTIONS).concat(expanded),
+      punctuation: DELIMIT
     },
     contains: [
       hljs.C_LINE_COMMENT_MODE,
       INCLUDE,
       hljs.HASH_COMMENT_MODE,
       BLOCK_COMMENT,
+     {
+       className: 'built_in',
+       variants: [
+         {
+          begin: /(?<=~)\s+(\w+)/
+         },
+         {
+          begin: /(?<=\+=)\s+(\w+(_lpdf|_lupdf|_lpmf|_cdf|_lcdf|_lccdf|_qf))/
```

Comment:
Don't we already have a manual list of these?

Reply:
Yes, but this is for user defined functions. We will not know what comes before for those. I will add a comment

Reply:
This is also another case of look-behind... I only mentioned it once, but you'll have to make sure NONE of your regex are using it across the entire grammar...

Reply:
Should the suffix version not also include the `(?=\())` lookahead filter?

Reply:
yes, you're right. Added back